src/semeio/forward_models/__init__.py

```diff
@@ -45,16 +56,20 @@ class DesignKW(ForwardModelStepPlugin):
     def __init__(self):
         super().__init__(
             name="DESIGN_KW",
-            command=["design_kw", "<template_file>", "<result_file>"],
+            command=["design_kw", "<template_file>", "<result_file>", "<VALIDATE>"],
+            default_mapping={"<VALIDATE>": "false"},
         )
 
     def validate_pre_realization_run(
         self, fm_step_json: ForwardModelStepJSON
     ) -> ForwardModelStepJSON:
         return fm_step_json
 
+    @validation_enabled
```

Comment:
Now when we won't do `DESIGN2PARAMS` validation maybe this can be simplified to the previous version? Although this looks quite nice :)